Index: app/models/model_news.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/models/model_news.php	(revision 8408db6f19da905bec03438cf1b11745ac69e5d6)
+++ app/models/model_news.php	(revision )
@@ -260,6 +260,7 @@
                                                     <a href="news/' . $result['n_id'] . '"
                                                        title="' . $result['title'] . '" class="image__linker">
                                                         <img src="public/images/news/' . $result['i_image'] . '"
+                                                             onerror="this.src=\'public/images/default.png\'"
                                                              width="284" height="180" class="image__img wp-post-image"
                                                              alt="' . $result['title'] . '"> </a>
 
